package.json

```diff
@@ -39,14 +39,14 @@
   },
   "scripts": {
     "build": "gulp",
-    "feature-tests": "./node_modules/.bin/cucumber-js test/feature --format=progress",
+    "feature-tests": "cucumber-js test/feature --format=progress",
```

Comment:
npm automatically adds `./node_modules/.bin` to the path of the shell that scripts run in. This does nothing but clutter up package.json